wordpress-mobile/WordPress-Editor-iOS
# Removes the Pods folder from the repository.

Since CocoaPods has become very fast at installing Pods from scratch, we're removing them from our repository.

**How to test:**

1. Make sure the changes are fine.
2. Do `pod install` and make sure the Pods folder is properly ignored.
